### PR TITLE
Remove unused events, interfaces, and patterns from GGVPointer

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview
This is largely a formatting / cleanup pass on GGVPointer, found as a result of my work on #5528.
Mostly, there was an interface that had entirely empty events. It also had a `Start` method that only called the base.

## Verification
1. Make sure GGVPointer still works.

### CODE REVIEW

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs [hunk 1]
```diff
@@ -9,24 +9,26 @@
 
 namespace Microsoft.MixedReality.Toolkit.Input
 {
-    public class GGVPointer : InputSystemGlobalHandlerListener, IMixedRealityPointer, IMixedRealityInputHandler, IMixedRealityInputHandler<MixedRealityPose>, IMixedRealitySourcePoseHandler, IMixedRealitySourceStateHandler
+    public class GGVPointer : InputSystemGlobalHandlerListener,
```
Comment (Troy-Ferrell) [inline Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs]: summary?

Comment (keveleigh) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/GGVPointer.cs]: Good catch!

**Pull request merged**